Orange/widgets/regression/owsgdregression.py

```diff
@@ -42,7 +40,7 @@ class OWSGDRegression(OWProvidesLearner, widget.OWWidget):
     learning_rate = settings.Setting(InvScaling)
 
     want_main_area = False
-    resizing_enabled = False
+    resizing_enabled = True
```

Comment:
No problem, that's what code review is for. But if you do opt to relayout, please do it in a separate PR. :smiley: